docs/src/pages/[platform]/theming/style-attributes/examples/DesignTokenTable.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import {
+  Expander,
+  ExpanderItem,
+  useTheme,
+  Table,
+  TableRow,
+  TableBody,
+  TableCell,
+  TableHead,
+  Tabs,
+  TabItem,
+} from '@aws-amplify/ui-react';
+
+const flattenTokens = (tokenObject) => {
+  if (tokenObject.value) {
+    return [tokenObject];
+  } else {
+    return Object.values(tokenObject).reduce((prev: [], current) => {
+      return [...prev, ...flattenTokens(current)];
+    }, []);
+  }
+};
+
+const convertValueToString = (value) => {
+  if (typeof value === 'string') {
+    return value;
+  }
+  return JSON.stringify(value);
+};
+
+const createExpander = (designObject) => {
```

Comment:
Except for `backgroundColor` and `color`, none of the other style properties have a Theme Root. Perhaps we can remove the Theme Root column altogether?

<img width="292" alt="Screen Shot 2022-06-13 at 11 12 39 AM" src="https://user-images.githubusercontent.com/48109584/173402122-4bd8de44-cb80-411e-b261-130206f64a81.png">

Reply:
+1, let's remove for now and revisit this column whe we have everything categorized

Reply:
removed